floodviz/hydrograph_utils.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+import operator
+import time
+
+
+def parse_hydrodata(jdata, newd, idx, sites_value_maxes):
+    siteName = jdata['value']['timeSeries'][0]['sourceInfo']['siteName']
```

Comment:
Rather than passing in the whole jdata dictionary, consider passing in just jdata['value']'[timeSeries'][0]